astrokat/noisediode.py

```diff
@@ -1,10 +1,15 @@
+from __future__ import division
+from __future__ import absolute_import
+
 import numpy as np
 import time
+
 libnames = ['user_logger']
+
 try:
-    lib = __import__('katcorelib', globals(), locals(), libnames, -1)
+    lib = __import__('katcorelib', globals(), locals(), libnames)
 except ImportError:
-    lib = __import__('simulate', globals(), locals(), libnames, -1)
+    lib = __import__('astrokat.simulate', globals(), locals(), libnames)
```

Comment:
Can't we drop the `astrokat` prefix here, so just `.simulate`?

(Also wonder if there is a better way to do these imports - it isn't pretty)